models/config_file_tree.go

```diff
@@ -2,11 +2,20 @@ package models
 
 import (
 	"fmt"
+
 	"gopkg.in/yaml.v2"
 )
 
-func (ymlTree *ConfigFileTreeModel) Merge() (string, error) {
-	result, err := merge(ymlTree)
+type ConfigFileTreeModel struct {
+	Path     string                `json:"path" yaml:"path"`
+	Contents string                `json:"contents,omitempty" yaml:"contents,omitempty"`
+	Includes []ConfigFileTreeModel `json:"includes,omitempty" yaml:"includes,omitempty"`
+
+	Depth int `json:"-" yaml:"-"`
```

Comment:
Do we need this depth field? It seems like we save the depth in it but it is not used in a meaningful way. And it is ignored in both the json and yaml serialisation.

Reply:
Good catch, thanks, this is not needed, removed in https://github.com/bitrise-io/bitrise/pull/982/commits/5d2cc065cbe9fd007c10b7ffd8d119c94f758fee